libtelemetry/src/main/java/com/mapbox/android/telemetry/TelemetryService.java

```diff
@@ -181,6 +179,16 @@ boolean isTelemetryReceiverRegistered() {
     return isTelemetryReceiverRegistered;
   }
 
+  boolean locationPermissionCheck() {
```

Comment:
if we're changing method's visibility modifier in order to get it under test harness, we should always annotate it with `@VisibleForTesting` otherwise someone may think that we're not properly encapsulating our code. Now, another question is it really necessary to change its visibility in order to test it?